pkg/utils/responses/errors.go

```diff
@@ -12,7 +12,7 @@ var (
 	ErrMemberNotFound            = "Team member with the specified ID was not found."
 	ErrMissingOrganizationName   = "Organization name is required."
 	ErrMissingOrganizationDomain = "Organization domain is required."
-	ErrDomainAlreadyExists       = "An organization with that domain already exists."
+	ErrDomainAlreadyExists       = "An organization with that workspace URL already exists."
```

Comment:
This minor change should prevent user confusion since the UI lists workspace URL instead of domain. Patrick said it would be ok to make this change.